backend/spherre/tests/test_service/test_transaction_service.py

```
@@ -0,0 +1,352 @@
+        # Test rejecting by a valid member
+        updated_transaction = TransactionService.reject_transaction(
+            transaction_id=1,
+            account=self.account,
+            member=self.member2
+        )
+        
+        assert updated_transaction is not None
+        assert self.member2 in updated_transaction.rejected.all()
+        
+        # Test rejecting a transaction that's already executed (should fail)
+        transaction2 = TransactionService.create_transaction(
```

Ruff (0.11.9)

Local variable `transaction2` is assigned to but never used

Remove assignment to unused variable `transaction2`

(F841)